### PR TITLE
Morphed add command to include workers and bodies.

### PR DESCRIPTION
**Pull request by shaoyi1997**
Add command for Person retained temporarily.

### CODE REVIEW

File: src/main/java/seedu/address/model/entity/worker/Worker.java [hunk 1]
```diff
@@ -115,15 +117,23 @@ public void setEmploymentStatus(String employmentStatus) {
...
         }
+    }
 
-        return otherWorker != null
-            && otherWorker.getName().equals(getName())
-            && otherWorker.getSex().equals(getSex())
-            && (otherWorker.getPhone().equals(getPhone()));
+    @Override
+    public boolean isSameEntity(Object o) {
+        return isSameEntity(o);
```
Comment (ambervoong) [inline src/main/java/seedu/address/model/entity/worker/Worker.java]:
```suggestion
        return isSamePerson(o);
```

Comment (ambervoong) [reply src/main/java/seedu/address/model/entity/worker/Worker.java]: Causes infinite loop.

Comment (arjavibahety) [inline src/main/java/seedu/address/model/entity/worker/Worker.java]: infinite recursion! >.<
Should it be `return o == this` ?

File: src/test/java/seedu/address/testutil/TypicalWorkers.java [hunk 2]
```diff
@@ -8,10 +11,10 @@
 public class TypicalWorkers {
 
     public static final Worker ALICE = new WorkerBuilder().withName("Alice Pauline")
```
Comment (arjavibahety) [inline src/test/java/seedu/address/testutil/TypicalWorkers.java]: Can we change ALICE to some other name? Or create another female worker? Amber is already using ALICE and for  `DeleteCommand` I need to test for both `Body` and `Worker`. So I cannot have 2 ALICES :( 
In my `Morph-Delete-Command` branch, I have added another worker

**Pull request merged**